### PR TITLE
[fact] Replace str.format with f-strings in GitLab module

### PR DESCRIPTION
**Pull request by ShreyaDhananjay**
#460
In [gitlab module](https://github.com/repobee/repobee/blob/master/src/_repobee/ext/gitlab.py)

### CODE REVIEW

File: src/_repobee/ext/gitlab.py [hunk 1]
```diff
@@ -187,15 +185,11 @@ def _ensure_members_in_team(
...
-                    ", ".join(missing_members), team.name
-                )
+                f'Adding members {", ".join(missing_members)} to team {team.name}'
             )
         if existing_members:
             LOGGER.info(
-                "{} already in team {}, skipping...".format(
-                    ", ".join(existing_members), team.name
-                )
+                f'{", ".join(existing_members)} already in team {team.name}, skipping...'
```
Comment (slarse) [inline src/_repobee/ext/gitlab.py:192]: Also just slightly too long.
```suggestion
                f'{", ".join(existing_members)} already in team {team.name}, '
                f"skipping..."
```

File: src/_repobee/ext/gitlab.py [hunk 2]
```diff
@@ -338,19 +330,15 @@ def get_repo_urls(
...
                 for repo_name in master_repo_names
             ]
             if not teams
             else [
-                "{}/{}/{}.git".format(
-                    group_url,
-                    team,
-                    plug.generate_repo_name(str(team), master_repo_name),
-                )
+                f"{group_url}/{team}/{plug.generate_repo_name(str(team), master_repo_name)}.git"
```
Comment (slarse) [inline src/_repobee/ext/gitlab.py:341]: This line is quite a bit too long.

```suggestion
                f"{group_url}/{team}/
                f"{plug.generate_repo_name(str(team), master_repo_name)}.git"
```

File: src/_repobee/ext/gitlab.py [hunk 3]
```diff
@@ -672,25 +652,23 @@ def _verify_group(group_name: str, gl: gitlab.Gitlab) -> None:
...
             if group.path == group_name
         ]
         if not slug_matched:
             raise exception.NotFoundError(
-                "Could not find group with slug {}. Verify that you have "
-                "access to the group, and that you've provided the slug "
-                "(the name in the address bar).".format(group_name)
+                f"Could not find group with slug {group_name}. Verify that you have "
+                f"access to the group, and that you've provided the slug "
+                f"(the name in the address bar)."
```
Comment (slarse) [inline src/_repobee/ext/gitlab.py:665]:
```suggestion
                f"Could not find group with slug {group_name}. Verify that "
                f"you have access to the group, and that you've provided "
                f"the slug (the name in the address bar)."
```

**Comment by ShreyaDhananjay** [comment]
Thanks for the feedback! I'll make the changes

**Comment by slarse** [comment]
@ShreyaDhananjay Thanks for making the changes, I will review the PR again as soon as I have some time on my hands :)

**Comment by ShreyaDhananjay** [comment]
Yes, thank you!

**Comment by slarse** [comment]
@ShreyaDhananjay Awesome, thanks a lot for your contribution!

**Pull request merged**